CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `conjurctl server` allows the operator to specify the admin user's password
+  via a file or STDIN using the optional `--password-file` flag.
+  [cyberark/conjur#2043](https://github.com/cyberark/conjur/issues/2043)
+- `conjurctl account` allows the operator to specify the admin user's password
+  via a file or STDIN using the optional `password_file` command argument.
+  [cyberark/conjur#2043](https://github.com/cyberark/conjur/issues/2043)
```

Comment:
duplicate?

Reply:
One is for the `conjurctl server` and the other is for the `conjurctl account` command. However, I will be changing both to use the flag `--password-file` and then I will be able to merge these two together